R/cut_dataToWaterYear.R

```diff
@@ -0,0 +1,48 @@
+#' Cuts the discharge time series to full water years
```

Comment:
Is this function called in any other functions? I didn't see it called anywhere when I searched.

Reply:
No, it is called nowhere. Initially I thought it could be called if the error occurs in the "fullYearCheck" in function "validate_data.R". But then I figured that it is better to leave the decision with the user on how to treat the missing data (e.g., if just a few days are missing, it may be more meaningful to estimate/fill the gaps rather than excluding a full year). 

So, I suggest to leave the function standalone and provide it as a convenient option to cut the data?